packages/salesforcedx-apex-debugger/test/unit/adapter/apexDebug.test.ts

```diff
@@ -1058,6 +1087,10 @@ describe('Interactive debugger adapter - unit', () => {
       runSpy.restore();
     });
 
+    afterAll(() => {
```

Comment:
there are a bunch of describe blocks in this test.  I'm attempting here to clear the pending timers as jest was complaining about open handles.